src/components/TodoList/TodoList.tsx

```diff
@@ -25,7 +40,52 @@ export const TodoList: React.FC = () => {
         </thead>
 
         <tbody>
-          <tr data-cy="todo">
+          {filteredTodos.map(todo => (
+            <tr
+              data-cy="todo"
+              key={todo.id}
+              {...(currentTodo &&
+                currentTodo.id === todo.id && {
+                  className: 'has-background-info-light',
+                })}
```

Comment:
Try to think how we can not use spread in attributes ( not critical)

Reply:
I am not sure I understand what I have to do?

You want me to make it like this: className: `${currentTodo && currentTodo.id === todo.id && 'has-background-info-light'}`?

Reply:
Spreading parameters is not the best idea here - as you have only one parameter (I avoid using it at all)

As long as you only have the `className` to set, you can simply use `classnames`

```
    className = {classnames({'has-background-info-light' : currentTodo?.id === todo.id)}
```

Notice, `?.` also performed `undefined`/`null` check